.changelog/unreleased/breaking-changes/1271-supersede-Bytes-with-cosmwasm-Binary.md

```diff
@@ -0,0 +1,2 @@
+- [ibc-client-cw] Supersede `Bytes` with `cosmwasm::Binary`.
+  ([\#1271](https://github.com/cosmos/ibc-rs/issues/1271))
```

Comment:
One last comment. Why do you think this is a breaking change? The way I see, the users of our cw-context shouldn't care about these changes. For example, we didn't need to touch our Tendermint light client cw contract.

Reply:
Ah, that was left over from the first commit, as I had touched ibc-client-wasm-types as well. fixed [9515258](https://github.com/cosmos/ibc-rs/pull/1272/commits/9515258ae09a79cf68d58ea428024549b2583612)